docs/docs/upgrading-node-js.md

```diff
@@ -35,7 +35,7 @@ There are multiple ways to update your version of Node.js depending on how you o
 
 This is the recommended way to install a newer version of Node.
 
-You will have Homebrew installed on your computer if you [followed part zero of the Gatsby tutorial](https://www.gatsbyjs.org/tutorial/part-zero/#-install-nodejs-and-npm). Homebrew is a program that allows you to install specific versions of Node.js (and other software).
+You will have Homebrew installed on your computer if you [followed part zero of the Gatsby tutorial](https://www.gatsbyjs.org/tutorial/part-zero/#install-nodejs-for-your-appropriate-operating-system). Homebrew is a program that allows you to install specific versions of Node.js (and other software).
```

Comment:
```suggestion
You will have Homebrew installed on your computer if you [followed part zero of the Gatsby tutorial](/tutorial/part-zero/#install-nodejs-for-your-appropriate-operating-system). Homebrew is a program that allows you to install specific versions of Node.js (and other software).
```

It looks like this could be a relative link to take advantage of Gatsby links, might as well change this while we're at it 🙂